models/articles.models.js

```diff
@@ -12,12 +12,19 @@ exports.fetchArticle = (article_id) => {
         return rows[0];
     });
 };
-
+exports.commentCount = (article_id) => {
+    return db.query(
+        `SELECT COUNT(*) FROM comments
```

Comment:
If we want the number of comments to be referred to as comment_count we also need to change something in here. Have a look at the AS keyword.